ansible/roles/erwin-idm/templates/secret.yml.j2

```
@@ -0,0 +1,15 @@
+#jinja2: trim_blocks: "True", lstrip_blocks: "True"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: erwinidm-secret
+  labels:
+    app: erwinidm
+  namespace: {{ NAMESPACE }}
+type: Opaque
+data:
+  KEYCLOAK_ADMIN: {{ item.ERWINIDM_ADMIN_USER | b64encode }}
+  KEYCLOAK_ADMIN_PASSWORD: {{ item.ERWINIDM_ADMIN_PASSWORD | b64encode }}
+  KC_DB_URL: {{ item.ERWINIDM_DB_URL | b64encode }}
+  KC_DB_USERNAME: {{ item.ERWINIDM_DB_USER | b64encode }}
+  KC_DB_PASSWORD: {{ item.ERWINIDM_DB_PASSWORD | b64encode }}
```

the **item** was something old where the config was other stored for the instances.
```suggestion
  KEYCLOAK_ADMIN: {{ ERWINIDM_ADMIN_USER | b64encode }}
  KEYCLOAK_ADMIN_PASSWORD: {{ ERWINIDM_ADMIN_PASSWORD | b64encode }}
  KC_DB_URL: {{ ERWINIDM_DB_URL | b64encode }}
  KC_DB_USERNAME: {{ ERWINIDM_DB_USER | b64encode }}
  KC_DB_PASSWORD: {{ ERWINIDM_DB_PASSWORD | b64encode }}
```

---

done